applications/GeoMechanicsApplication/tests/cpp_tests/custom_conditions/test_axisymmetric_U_Pw_normal_face_load_condition.cpp

```diff
@@ -0,0 +1,62 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#include "containers/model.h"
+#include "custom_conditions/axisymmetric_U_Pw_normal_face_load_condition.hpp"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+
+#include <geometries/line_2d_4.h>
+#include <geometries/line_2d_5.h>
```

Comment:
I think it is better to reserve the '<>' for system includes (e.g. include <vector>) and keep this as ""
```suggestion
#include "geometries/line_2d_4.h"
#include "geometries/line_2d_5.h"
```

Reply:
Thank you, Richard. As far as I understand CLion adds includes with <>. Just in case I changed some <> in the tests\cpp_tests folder.

Reply:
Indeed, there's probably a setting somewhere that we could set (I haven't looked for it yet) such that CLion uses the one we prefer.